src.js

```diff
@@ -91,6 +128,19 @@ function createCanvasCircle(ctx, opts = {
   ctx.fill(path);
 }
 
+function createCanvasVirus(ctx, opts = {
```

Comment:
This is identical to `createCanvasCircle`, you should remove it and just use `createCanvasCircle`.